### PR TITLE
Fixed an issue where buttons in BottombarButtons (BottomSheet) did not take up full width

### PR DESCRIPTION
**Pull request by Vetle444**
### Todos
- [X] I have tested on an Android device.
- [X] I have tested on an iOS device.
- [ ] I have supported accessibility

### CODE REVIEW

File: src/library/DIPS.Mobile.UI/Components/BottomSheets/Android/BottomSheetHandler.cs [hunk 1]
```diff
@@ -95,7 +95,7 @@ public partial class BottomSheetHandler : ContentViewHandler
         {
             m_bottomBar = (ContentViewGroup)m_bottomSheet.CreateBottomBar().ToPlatform(MauiContext!);
             m_bottomBar.LayoutParameters =
-                new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MatchParent, ViewGroup.LayoutParams.WrapContent);
+                new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MatchParent, (int)Context.ToPixels(120));
```
Comment (haavamoa) [inline src/library/DIPS.Mobile.UI/Components/BottomSheets/Android/BottomSheetHandler.cs:98]: hardkode 120?

**Review by haavamoa — CHANGES_REQUESTED** [review]

File: src/library/DIPS.Mobile.UI/Components/BottomSheets/Android/BottomSheetHandler.cs (hunk 1 above)
Comment (Vetle444) [reply src/library/DIPS.Mobile.UI/Components/BottomSheets/Android/BottomSheetHandler.cs:98]: Ja som du hardkoda i CreateBottomBar() 🤪

Comment (haavamoa) [reply src/library/DIPS.Mobile.UI/Components/BottomSheets/Android/BottomSheetHandler.cs]: Ah glem det, jeg leste jo helt feil. Dette var høyde :P Ja, det stemmer at den var hardkodet

Comment (Vetle444) [reply src/library/DIPS.Mobile.UI/Components/BottomSheets/Android/BottomSheetHandler.cs]: Dettan e høgda

**Review by haavamoa — APPROVED** [review]

**Pull request merged**